### PR TITLE
Translate "DoS vulnerabilities in REXML" (ko)

### PR DESCRIPTION
**Pull request by riseshia**
:link: https://github.com/ruby/www.ruby-lang.org/issues/2818

Translates #3332, #3333

actual diff is 5bbc3e399835d86a3d5de4f5ce9c6e2d498b090f

### CODE REVIEW

File: ko/news/_posts/2024-08-01-dos-rexml-cve-2024-41123.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+layout: news_post
+title: "CVE-2024-41123: REXML의 DoS 취약점"
+author: "kou"
+translator: "shia"
+date: 2024-08-01 03:00:00 +0000
+tags: security
+lang: ko
+---
+
+REXML gem에서 DoS 취약점이 발견되었습니다. 이 취약점은 CVE 번호 [CVE-2024-41123](https://www.cve.org/CVERecord?id=CVE-2024-41123)로 등록되었습니다. REXML gem 업그레이드를 강하게 추천합니다.
```
Comment (yous) [inline ko/news/_posts/2024-08-01-dos-rexml-cve-2024-41123.md:11]:
```suggestion
REXML gem에서 DoS 취약점이 몇 개 발견되었습니다. 이 취약점은 CVE 번호 [CVE-2024-41123](https://www.cve.org/CVERecord?id=CVE-2024-41123)으로 등록되었습니다. REXML gem 업그레이드를 강하게 추천합니다.
```

File: ko/news/_posts/2024-08-01-dos-rexml-cve-2024-41946.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+layout: news_post
+title: "CVE-2024-41946: REXML의 DoS 취약점"
+author: "kou"
+translator: "shia"
+date: 2024-08-01 03:00:00 +0000
+tags: security
+lang: ko
+---
+
+REXML gem에서 DoS 취약점이 발견되었습니다. 이 취약점은 CVE 번호 [CVE-2024-41946](https://www.cve.org/CVERecord?id=CVE-2024-41946)로 등록되었습니다. REXML gem 업그레이드를 강하게 추천합니다.
```
Comment (yous) [inline ko/news/_posts/2024-08-01-dos-rexml-cve-2024-41946.md:11]:
```suggestion
REXML gem에서 DoS 취약점이 발견되었습니다. 이 취약점은 CVE 번호 [CVE-2024-41946](https://www.cve.org/CVERecord?id=CVE-2024-41946)으로 등록되었습니다. REXML gem 업그레이드를 강하게 추천합니다.
```

File: ko/news/_posts/2024-08-01-dos-rexml-cve-2024-41946.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## 세부 내용
+
+SAX2나 pull parser API로 많은 엔티티 확장을 포함하는 XML을 파싱할 때, REXML gem은 처리에 긴 시간이 걸립니다.
```
Comment (yous) [inline ko/news/_posts/2024-08-01-dos-rexml-cve-2024-41946.md:15]: How about this? It seems there are pull parser and push parser: <https://stackoverflow.com/questions/15895124/what-is-push-approach-and-pull-approach-to-parsing>.

```suggestion
SAX2나 풀(pull) 파서 API로 많은 엔티티 확장을 포함하는 XML을 파싱할 때, REXML gem은 처리에 긴 시간이 걸립니다.
```

Comment (riseshia) [reply ko/news/_posts/2024-08-01-dos-rexml-cve-2024-41946.md]: fixed e3be39e

File: ko/news/_posts/2024-08-01-dos-rexml-cve-2024-41946.md (hunk 2 above)
Comment (riseshia) [reply ko/news/_posts/2024-08-01-dos-rexml-cve-2024-41946.md]: fixed e3be39e

File: ko/news/_posts/2024-08-01-dos-rexml-cve-2024-41123.md (hunk 1 above)
Comment (riseshia) [reply ko/news/_posts/2024-08-01-dos-rexml-cve-2024-41123.md]: fixed e3be39e

**Review by yous — APPROVED** [review]
Thanks!

**Pull request merged**